README.md

```diff
@@ -430,7 +458,13 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to describe index \"%s\". Error:%s", idx.Name, err)
 	} else {
-		fmt.Printf("Successfully found the \"%s\" index!\n", idx.Name)
+		desc := fmt.Sprintf("Description: \n  Name: %s\n  Dimension: %d\n  Host: %s\n  Metric: %s\n"+
+			"  DeletionProtection"+
+			": %s\n"+
+			"  Spec: %+v"+
+			"\n  Status: %+v\n",
+			idx.Name, idx.Dimension, idx.Host, idx.Metric, idx.DeletionProtection, idx.Spec, idx.Status)
```

Comment:
Should `*idx.Spec` be printed, rather than `idx.Spec`?

Similarly, `*idx.Status` rather than `idx.Status`?

Reply:
I believe using `%+v` will automatically print the pointer, but I'm actually not positive. You may be right.

Reply:
There's no difference (in the output) between printing out `idx.Spec` vs `*idx.Spec`. Right now, the output looks like this (I think drilling down to `Pod` or `Serverless` within `Spec` is too ugly): 

<img width="526" alt="Screenshot 2024-08-02 at 2 13 06 PM" src="https://github.com/user-attachments/assets/474e2815-6f3d-4b6e-aa75-625132785ce5">

Reply:
The '*' gets rid of the '&' in front of the struct in Spec and Status